spec/controllers/projects_controller_spec.rb

```diff
@@ -16,7 +16,9 @@
     end
 
     context "a signed in user" do
-      let(:user) { FactoryBot.create :user }
+      # TODO: Switch for a user with less access than system admin
```

Comment:
I force the user to be a sysadmin so that the test works as it did before we had a check for access to the project. We should eventually update these tests to use a more restrictive user (but with access to the project), hence the TODO.